index.js

```diff
@@ -22,7 +22,11 @@ exports.init = function(opts) {
   exports.SchemaAdapter = require('./adapters/mongoose');
 
   exports.getModels = function () {
-    return opts.mongoose.models;
+    if ((typeof opts.getModels) === 'function') {
```

Comment:
An option named `included_models` containing an array of model names would be great instead of a `getModels` option.